index.d.ts

```diff
@@ -23,6 +24,8 @@ declare namespace tempy {
 			readonly name?: string;
 		}
 	>;
+
+	type WritableType = string | Buffer | TypedArray | DataView
```

Comment:
This type makes no sense as it's not the writable type for the whole module, it's just for the `write()` method and not `writeSync()`. Better to just inline the type.

Reply:
Fixed!